development/docker-compose.dev.yml

```diff
@@ -24,3 +24,19 @@ services:
     volumes:
       - "./nautobot_config.py:/opt/nautobot/nautobot_config.py"
       - "../:/source"
+  mattermost:
+    image: "mattermost/mattermost-preview"
+    hostname: "mattermost"
+    environment:
+      MM_SERVICESETTINGS_ALLOWEDUNTRUSTEDINTERNALCONNECTIONS: "0.0.0.0/0"
+      MM_SERVICESETTINGS_ENABLEDEVELOPER: "true"
+      MM_SERVICESETTINGS_SITEURL: "http://mattermost:8065"
+      MM_TEAMSETTINGS_ENABLEOPENSERVER: "true"
+    ports:
+      - "8065:8065"
+    volumes:
+      - "./mattermost/dump.sql:/docker-entrypoint-initdb.d/dump.sql"
+      - "mattermost-data:/var/lib/mysql"
+
+volumes:
+  mattermost-data: {}
```

Comment:
Let's move this out to its own compose file and mimic what Core uses for SSO: https://github.com/nautobot/nautobot/blob/develop/development/docker-compose.keycloak.yml

This allows adding a single line into `invoke.yml`: https://github.com/nautobot/nautobot/blob/bdf1faa6386a0fa866978f6eb866413a336cb9f9/invoke.yml.example#L37

Then the environment variable set in `dev.env` can be automatically applied to the right containers.

Reply:
Applied.

I moved mattermost compose configuration to `development/mattermost/docker-compose.yml`, WDYT @bryanculver ?